Makefile

```diff
@@ -120,13 +120,13 @@ uninstall:
 coverage: ## generate coverprofiles from the unit tests
 	@echo "🐳 $@"
 	@( for pkg in $(filter-out ${INTEGRATION_PACKAGE},${PACKAGES}); do \
-		go test -i -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile="../../../$$pkg/coverage.txt" -covermode=atomic $$pkg || exit; \
-		go test -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile="../../../$$pkg/coverage.txt" -covermode=atomic $$pkg || exit; \
+		go test -i -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile=coverage.txt -covermode=atomic $$pkg || exit; \
+		go test -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile=coverage.txt -covermode=atomic $$pkg || exit; \
 	done )
 
 coverage-integration: ## generate coverprofiles from the integration tests
 	@echo "🐳 $@"
-	go test -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile="../../../${INTEGRATION_PACKAGE}/coverage.txt" -covermode=atomic ${INTEGRATION_PACKAGE}
+	go test -race -tags "${DOCKER_BUILDTAGS}" -test.short -coverprofile=coverage.txt -covermode=atomic ${INTEGRATION_PACKAGE}
```

Comment:
```coverage-integration``` is not called in ```.travis.yml```. I just made the change to be consistent (of sort ...). When ```integration``` becomes available, probably I joint CI target is to be created (to allow appending to ```coverage.txt```